msglibrary/src/test/java/ingsw/group1/msglibrary/RandomSMSPeerGeneratorTest.java

```diff
@@ -0,0 +1,72 @@
+package ingsw.group1.msglibrary;
+
+import android.util.ArraySet;
+
+import com.google.i18n.phonenumbers.PhoneNumberUtil;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+import java.util.Set;
+
+import static junit.framework.TestCase.assertNotSame;
+import static junit.framework.TestCase.assertSame;
+import static junit.framework.TestCase.assertTrue;
+
+@RunWith(Parameterized.class)
+public class RandomSMSPeerGeneratorTest {
+
+    private static List<String> COUNTRIES = new ArrayList<>();
+
+    private RandomSMSPeerGenerator generator = new RandomSMSPeerGenerator();
+
+    private final int run;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'run' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-message-library/pullRequest?prid=4741620)